docs/developer/extending/apps/extending-dashboard-with-apps.mdx

```diff
@@ -51,17 +51,44 @@ The example manifest below defines two extensions, one providing a custom produc
 
 Saleor requires extensions to define a mounting place. The table below explains all mounting locations currently supported by Saleor.
 
-|             Mount             |                              Description                               |
-| :---------------------------: | :--------------------------------------------------------------------: |
-| PRODUCT_DETAILS_MORE_ACTIONS  | Mount extension on product's detail page under the more action button. |
-|    PRODUCT_OVERVIEW_CREATE    |    Mount extension on product's list page under the create button.     |
-| PRODUCT_OVERVIEW_MORE_ACTIONS |  Mount extension on product's list page under the more action button.  |
-|      NAVIGATION_CATALOG       |      Mount extension in `Catalogs` section in the navigation bar.      |
-|       NAVIGATION_ORDERS       |       Mount extension in `Orders` section in the navigation bar.       |
-|     NAVIGATION_CUSTOMERS      |     Mount extension in `Customers` section in the navigation bar.      |
-|     NAVIGATION_DISCOUNTS      |     Mount extension in `Discounts` section in the navigation bar.      |
-|    NAVIGATION_TRANSLATIONS    |    Mount extension in `Translations` section in the navigation bar.    |
-|       NAVIGATION_PAGES        |       Mount extension in `Pages` section in the navigation bar.        |
-|  ORDER_DETAILS_MORE_ACTIONS   |  Mount extension on order's detail page under the more action button.  |
-|     ORDER_OVERVIEW_CREATE     |     Mount extension on order's list page under the create button.      |
-|  ORDER_OVERVIEW_MORE_ACTIONS  |   Mount extension on order's list page under the more action button.   |
+|                Mount                |                                 Description                                 |
+| :---------------------------------: | :-------------------------------------------------------------------------: |
+|       CATEGORY_OVERVIEW_CREATE      |       Mount extension on categories list page under the create button.     |
```

Comment:
Suggestion: this is a repetive thing, maybe we can drop "mount extension on" and make it less verbose